packages/systemd/Cargo.toml

```diff
@@ -26,3 +26,4 @@ libseccomp = { path = "../libseccomp" }
 libselinux = { path = "../libselinux" }
 libxcrypt = { path = "../libxcrypt" }
 util-linux = { path = "../util-linux" }
+
```

Comment:
nit: I don't think we want to add a newline to the end of the file. Can you check your editor settings?

Reply:
:/  That's weird - will fix